packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -32,12 +32,22 @@ abstract class Node {
   /// nodes in the tree. It may be [AffineMatrix.identity] to indicate that no
   /// additional transformation is needed.
   void build(DrawCommandBuilder builder, AffineMatrix transform);
+
+  /// Look up the bounds for the nearest sized parent.
+  Rect nearestParentBounds() => parent?.nearestParentBounds() ?? Rect.zero;
```

Comment:
Let's make this a `final Rect nearestParentBounds` and a required param in the ctor.

Reply:
Alternatively, this probably belongs on `AttributedNode`.